users/models.py

```diff
@@ -109,7 +109,7 @@ class CustomUser(AbstractUser):
     birthday = models.DateField(
         blank=False,
         verbose_name=_('Birthday'),
-        help_text=_('Format: DD.MM.YYYY'),
+        # help_text=_('Format: DD.MM.YYYY'),
```

Comment:
Remove comment, unless this needs to be returned.

Reply:
https://github.com/TampereHacklab/mulysa/pull/109/commits/039107ef48761d2457eb3511ec85b4271000b27f